manifests/charts/istio-telemetry/mixer-telemetry/templates/autoscale.yaml

```diff
@@ -1,6 +1,6 @@
 {{ $telemetry := index .Values "mixer" "telemetry" }}
 {{- if and $telemetry.autoscaleEnabled $telemetry.autoscaleMin $telemetry.autoscaleMax }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2beta2
```

Comment:
why not autoscaling/v1?

Reply:
I believe this is the reason:

>The beta version, which includes support for scaling on memory and custom metrics, can be found in autoscaling/v2beta2

As seen [here](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).

Reply:
Also, I assumed `v2beta2` would be more of an upgrade from `v2beta1` as opposed to `v1`

Reply:
oh of course. I was thinking v1 > v2beta2

Reply:
Cool! I'll address the e2e failures then :)